client/src/elm/Activity/Utils.elm

```diff
@@ -467,18 +467,7 @@ expectMotherActivity currentDate offlineSession motherId activity =
 
                                         entitledByUbudehe =
                                             Dict.get motherId offlineSession.mothers
-                                                |> Maybe.map
-                                                    (\mother ->
-                                                        case mother.ubudehe of
-                                                            Just Ubudehe1 ->
-                                                                True
-
-                                                            Just Ubudehe2 ->
-                                                                True
-
-                                                            _ ->
-                                                                False
-                                                    )
+                                                |> Maybe.map (\_ -> True)
```

Comment:
If there's no more dependency on Ubudehe, and every mother should get it, drop the condition.

Reply:
@Diama1 then drop the condition